templates/telemetry/health/health.4.schema.json

```diff
@@ -0,0 +1,95 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "health",
+  "type": "object",
+  "properties": {
+    @TELEMETRY_APPLICATION_1_JSON@,
+    @TELEMETRY_CLIENTID_1_JSON@,
+    @TELEMETRY_CREATIONDATE_1_JSON@,
+    @TELEMETRY_ID_1_JSON@,
+    "payload": {
+      "additionalProperties": false,
+      "properties": {
+        "os": {
+          "additionalProperties": false,
+          "properties": {
+            "name": {
+              "type": "string"
+            },
+            "version": {
+              "type": [
+                "string",
+                "integer"
+              ]
+            }
+          },
+          "type": "object"
+        },
+        "reason": {
+          "enum": [
+            "immediate",
+            "delayed",
+            "shutdown"
+          ],
+          "type": "string"
+        },
+        "sendFailure": {
+          "properties": {
+            "additionalProperties": {
+              "type": "number"
+            }
+          },
+          "minProperties": 1,
+          "type": "object"
+        },
+        "pingDiscardedForSize": {
+          "properties": {
+            "additionalProperties": {
+              "type": "number"
+            }
+          },
+          "minProperties": 1,
+          "maxProperties": 10,
```

Comment:
Why have this? Seems like a footgun that could hurt us as we expand the number of types of pings we are sending. (iiuc, this is limiting the number of ping types that can be included, as `pingDiscardedForSize` is a map from `pingType` -> `numberDiscarded`)